docs/models/anthropic.md

```diff
@@ -78,6 +78,106 @@ agent = Agent(model)
 ...
 ```
 
+## Cloud Platform Integrations
+
+You can use Anthropic models through cloud platforms by passing a custom client to [`AnthropicProvider`][pydantic_ai.providers.anthropic.AnthropicProvider].
+
+### AWS Bedrock
+
+To use Claude models via [AWS Bedrock](https://aws.amazon.com/bedrock/claude/):
+
+=== "With Pydantic AI Gateway"
+
+    ```python {title="<a href='/gateway/' style='float: right;'>Learn about Gateway</a>" test="skip"}
+    from pydantic_ai import Agent
+
+    agent = Agent('gateway/bedrock:us.anthropic.claude-haiku-4-5-20251001-v1:0')
+    ...
+    ```
+
+=== "Directly to Provider API"
+
+    Use the [`AsyncAnthropicBedrock`](https://docs.anthropic.com/en/api/claude-on-amazon-bedrock) client from the `anthropic` package:
+
+    ```python {test="skip"}
+    from anthropic import AsyncAnthropicBedrock
+
+    from pydantic_ai import Agent
+    from pydantic_ai.models.anthropic import AnthropicModel
+    from pydantic_ai.providers.anthropic import AnthropicProvider
+
+    bedrock_client = AsyncAnthropicBedrock()  # Uses AWS credentials from environment
+    provider = AnthropicProvider(anthropic_client=bedrock_client)
+    model = AnthropicModel('claude-sonnet-4-5', provider=provider)
+    agent = Agent(model)
+    ...
+    ```
+
+    !!! note "Bedrock vs BedrockConverseModel"
+        This approach uses Anthropic's SDK with AWS Bedrock credentials. For an alternative using AWS SDK (boto3) directly, see [`BedrockConverseModel`](bedrock.md).
+
+### Google Vertex AI
+
+To use Claude models via [Google Cloud Vertex AI](https://cloud.google.com/vertex-ai/generative-ai/docs/partner-models/use-claude):
+
+=== "With Pydantic AI Gateway"
+
+    ```python {title="<a href='/gateway/' style='float: right;'>Learn about Gateway</a>" test="skip"}
+    from pydantic_ai import Agent
+
+    agent = Agent('gateway/google-vertex:claude-sonnet-4-5@20250514')
```

Comment:
Same as up; this uses the Google Vertex API, rather than the Anthropic API hosted on Vertex.